time timestamp
sensor_msgs/Image image_left
sensor_msgs/Image image_mask
string image_path
string image_mask_path
